### ISSUE TITLE
Got Error "Error scanning stdout: bufio.Scanner: token too long"

### ISSUE THREAD

**Issue by jobamoorthy** [openfaas/of-watchdog#128]
Got Error "Error scanning stdout: bufio.Scanner: token too long" 
When we use easyocr python package -> https://github.com/JaidedAI/EasyOCR

## Expected Behaviour

Sample 1:
We are try this without openfass, easyocr is working fine. [refer attachment - easyocrDockerwithoutopenfass]
![image](https://user-images.githubusercontent.com/86336914/143384171-b58892bc-055b-4218-9808-5a9746707c55.png)

Sample 2:
we try with openfass, easyocr is not working got "Error scanning stdout: bufio.Scanner: token too long" [refer attachment - easyocrDockerwithopenfass] 

Referring https://www.openfaas.com/blog/openfaas-flask/  (Running an existing Flask app)
 

![image](https://user-images.githubusercontent.com/86336914/143384215-45a17009-3938-4e5a-b90c-284cb0d56470.png)

## Current Behaviour
Sample 1: 
   Working fine 

Sample 2: 
Give Error : "Error scanning stdout: bufio.Scanner: token too long" 

## Steps to Reproduce (for bugs)
basically this issue occurs when we use below line of code, which means initialize the easyocr reader

`reader = easyocr.Reader(['en'])`

## Context
Easy OCR model we would like to use using openfass

## Your Environment
Used docker version : Docker version 20.10.8, build 3967b7d
Python : 3.8.0 along with easyocr, falsk, etc., 

Kindly refer attached projects with docker file and requirement.txt and source code file
Basically the attached zip file contains two projects
1.easyocrDockerwithoutopenfass
2.easyocrDockerwithopenfass

[easyocrDocker.zip](https://github.com/openfaas/of-watchdog/files/7600530/easyocrDocker.zip)

**Comment by alexellis** [comment]
Hi @jobamoorthy thanks for your interest in OpenFaaS

Whilst we cannot debug your source code for you, we do offer consulting. Feel free to book a call with me if you'd like expert help https://calendly.com/alexellis

In the meantime, I would suggest that you use a recent version of the watchdog and read up on the README.md where it explains how to set a larger buffer for log lines. You also appear to be using a custom Dockerfile instead of our official template, this could also be causing you problems.

https://github.com/orgs/openfaas/packages/container/package/of-watchdog

Regards,

Alex
